### ISSUE TITLE
Set IG version to R4B

### ISSUE THREAD

**Issue by vadi2** [vadi2/MRI-Scheduling#1]
Need to change IG version from R4 to R4B - currently blocked by https://chat.fhir.org/#narrow/stream/215610-shorthand/topic/R4B.20sushi.20download.20doesn't.20work.

**Comment by vadi2** [comment]
Will put this on hold for now since ILCore packages are currently R4 only.
